packages/blockchain-wallet-v4-frontend/src/components/Form/SelectBoxBchAddresses/selectors.js

```diff
@@ -1,26 +1,26 @@
 import {
-  assoc,
-  assocPath,
   concat,
   compose,
   curry,
   descend,
   filter,
   has,
   head,
-  isNil,
   lensIndex,
   lensProp,
-  lift,
-  not,
   map,
   path,
   prepend,
   prop,
   reduce,
   set,
   sequence,
-  sort
+  sort,
+  lift,
+  assoc,
+  assocPath,
+  not,
+  isNil
```

Comment:
Interesting diff

Reply:
please keep these imported sorted by name

Reply:
I had taken those out previously when re-factoring the bch dropdown selector to more closely resemble the btc side. Realized I needed to filter out watch only addresses so had to put that functionality back.

Noted on alphabetical ordering of imports :  )